diff --git a/calamari_ocr/scripts/resume_training.py b/calamari_ocr/scripts/resume_training.py
index 1c9fe86..d181d16 100644
--- a/calamari_ocr/scripts/resume_training.py
+++ b/calamari_ocr/scripts/resume_training.py
@@ -36,6 +36,10 @@ def main():
     parser.add_argument("--no_skip_invalid_gt", action="store_true",
                         help="Do no skip invalid gt, instead raise an exception.")

+    # misc
+    parser.add_argument("--no_progress_bars", action="store_true",
+                        help="Do not show any progress bars")
+
     args = parser.parse_args()

     if args.gt_extension is None:
@@ -101,8 +105,8 @@ def main():
         trainer = Trainer(checkpoint_params, dataset,
                           validation_dataset=validation_dataset,
                           weights=args.checkpoint)
-        trainer.train(progress_bar=True)
+        trainer.train(progress_bar=not args.no_progress_bars)


 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
